src/app/clear-account/clear-account.component.html

```diff
@@ -0,0 +1,56 @@
+<app-banner></app-banner>
+
+<div
+  class="container home-container"
+  *ngIf="globalVars.inTab || globalVars.webview"
+>
+  <div class="font-weight-bold fs-30px mb-20px">Clear Login Data</div>
```

Comment:
I changed the verbiage here from 'wipe' --> 'clear' since I felt like it reads and looks better. I can certainly change it back if we want to stick with 'wipe', though.

Reply:
I think "Clear" is a better choice here. Let's stick with your change.